src/bitwarden_fmt_plug.c

```diff
@@ -0,0 +1,221 @@
+/*
+ * JtR format to crack Bitwarden hashes.
+ *
+ * This software is Copyright (c) 2018, Dhiru Kholia <kholia at kth.se> and it
+ * is hereby released to the general public under the following terms:
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted.
+ *
+ * Big thanks goes to Joshua Stein for making this format possible.
+ */
+
+#if FMT_EXTERNS_H
+extern struct fmt_main fmt_bitwarden;
+#elif FMT_REGISTERS_H
+john_register_one(&fmt_bitwarden);
+#else
+
+#include <string.h>
+
+#ifdef _OPENMP
+#include <omp.h>
+#ifndef OMP_SCALE
+#define OMP_SCALE               8
+#endif
```

Comment:
OMP_SCALE should be outside of the `#if _OPENMP`

Reply:
Is there a "reference format" I can refer to, so that I am following the latest best practices?

Reply:
#3091 says it all

Reply:
#3091 is awesome. I just followed it and got good results :+1: